index.html

```diff
@@ -55,31 +60,31 @@ <h1>Coronavirus Disease (COVID-19) Japan Tracker</h1>
         <td id="kpi-deceased"><div class="lds-dual-ring"></div></td>
       </tr>
       <tr class="kpi-labels">
-        <td>Confirmed</td>
-        <td>Recovered</td>
-        <td>Deceased</td>
+        <td data-ja="感染者数">Confirmed</td>
+        <td data-ja="回復者数">Recovered</td>
+        <td data-ja="死亡者数">Deceased</td>
       </tr>
     </table>
   </section>
 
   <a name="map"></a>
   <div class="center-offset">
-    <h4>Cases By Prefecture</h4>
+    <h4 data-ja="都道府県ごとの件数">Cases By Prefecture</h4>
```

Comment:
The Japanese is the direct translation from the English.
But here I think the English name should also change a bit, so it will differ from the "Prefecture Data"
Can we call it "Outbreak Map"? 
"ウイルス感染マップ”

Reply:
Updated